content/docs/tasks/telemetry/kiali/index.md

```diff
@@ -22,11 +22,21 @@ To install Kiali without using Helm, following the [Kiali install instructions](
 Create a secret in your Istio namespace with the credentials that you use to
 authenticate to Kiali. See the
 [Helm README](https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/README.md#installing-the-chart)
-for details. Modify and run the following example commands to create a secret:
+for details.
+
+First define the credentials you want to use for the Kiali username and passphrase:
+
+```bash
```

Comment:
Please use the {{< text bash >}} annotation here and elsewhere. You'll need to precede any command-line with a $

Reply:
> Please use the {{< text bash >}} annotation here and elsewhere. You'll need to precede any command-line with a $

@geeknoid I originally did try it with "text bash" when originally writing these docs, but I had to revert to using '''bash. Read the comments and see the screen snapshots in this PR comment to see why it is not using "text bash" (short answer: there is a bug in the text bash rendering of "here documents"): https://github.com/istio/istio.io/pull/2715#discussion_r223476239

Reply:
Well, now that I re-read your comment and WHAT line you are commenting on, I see you meant to make the two one-liners wrapped in text-bash! Sorry - I will fix that now.

Reply:
OK, this is done - see commit b3a94b48c93773ab65e4a5a220495db1d52bd3ab
My previous comment is still valid - we still need the one '''bash due to the "here document" not being rendered correctly inside a text-bash block.